contrib/drivers/filesharedrivers/nfsnative/nfsnative.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 The OpenSDS Authors.
```

Comment:
Change to contrib/drivers/filesharedrivers/nfs/nfs.go

Reply:
Done